docs/getting-started-monitoring.md

```diff
@@ -0,0 +1,294 @@
+# Getting Started with Helm Charts (Monitoring using Prometheus Operator)
+
+This document explains how to get started with monitoring Scalar products on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing.  
+
+## Environment
+
+In this guide, we will create the following environment in your local by using Docker and minikube.  
+
+```
++--------------------------------------------------------------------------------------------------+
+| +------------------------------------------------------+                     +-----------------+ |
+| |                kube-prometheus-stack                 |                     | Scalar Products | |
+| |                                                      |                     |                 | |
+| | +--------------+  +--------------+  +--------------+ | ---(Monitoring)---> |  +-----------+  | |
+| | |  Prometheus  |  | Alertmanager |  |   Grafana    | |                     |  | Scalar DB |  | |
+| | +-------+------+  +------+-------+  +------+-------+ |                     |  +-----------+  | |
+| |         |                |                 |         |                     |  +-----------+  | |
+| |         +----------------+-----------------+         |                     |  | Scalar DL |  | |
+| |                          |                           |                     |  +-----------+  | |
+| +--------------------------+---------------------------+                     +-----------------+ |
+|                            |                                                                     |
+|                            |         Kubernetes (minikube)                                       |
++----------------------------+---------------------------------------------------------------------+
+                             | <- expose to localhost (127.0.0.1)
+                             |
+              (Access Dashboard through HTTP)
+                             |
+                        +----+----+
+                        | Browser |
+                        +---------+
+```
+
+## Step 1. Install tools
```

Comment:
This section is a redundancy of the `Getting Started with Helm Charts (Scalar DB/DL)`.  
So, I will separate this section in the another PR after each guide are merged.